doc/xmlrpc_client.py

```diff
@@ -0,0 +1,21 @@
+import xmlrpc.client
```

Comment:
Try import like this

```suggestion
from xmlrpc import client as xmlrpclib
```